app/jobs/inventory_host_updated_job.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# Updates the host according to the inventory definition
+class InventoryHostUpdatedJob
+  include Sidekiq::Worker
+  include RecordNotFound
+
+  def perform(message)
+    if message['host'] && message['host']['display_name']
```

Comment:
There's a nicer way, if you want:

```suggestion
    if message.dig('host', 'display_name')
```